src/components/screens/monitor/blocks/blocks.js

```diff
@@ -2,94 +2,123 @@ import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import React from 'react';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { DEFAULT_LIMIT } from '../../../../constants/monitor';
 import { DateTimeFromTimestamp } from '../../../toolbox/timestamp';
+import BlockFilterDropdown from './blockFilterDropdown';
 import Box from '../../../toolbox/box';
 import Feedback from '../../../toolbox/feedback/feedback';
+import FilterBar from '../../../shared/filterBar';
 import LiskAmount from '../../../shared/liskAmount';
 import PageLayout from '../../../toolbox/pageLayout';
 import TableRow from '../../../toolbox/table/tableRow';
 import routes from '../../../../constants/routes';
+import styles from './blocks.css';
+import withFilters from '../../../../utils/withFilters';
 
-const columnClassNames = {
-  id: `${grid['col-md-2']} ${grid['col-xs-3']}`,
-  height: grid['col-xs-2'],
-  date: `${grid['col-md-2']} hidden-m`,
-  transactions: grid['col-xs-2'],
-  generator: grid['col-xs-2'],
-  amount: grid['col-xs-1'],
-  forged: `${grid['col-md-1']} ${grid['col-xs-2']}`,
-};
+const Blocks = ({
+  t, blocks, filters, applyFilters, clearFilter, clearAllFilters,
+}) => {
+  /* eslint-disable react/display-name */
+  const columns = [{
+    header: t('ID'),
+    className: `${grid['col-md-2']} ${grid['col-xs-3']} block-id`,
+    getValue: block => <Link to={`${routes.blocks.path}/${block.id}`}>{block.id}</Link>,
+  }, {
+    header: t('Height'),
+    className: grid['col-xs-2'],
+    getValue: block => block.height,
+  }, {
+    header: t('Date'),
+    className: `${grid['col-md-2']} hidden-m`,
+    getValue: block => <DateTimeFromTimestamp time={block.timestamp * 1000} token="BTC" />,
+  }, {
+    header: t('Transactions'),
+    className: grid['col-xs-2'],
+    getValue: block => block.numberOfTransactions,
+  }, {
+    header: t('Generated by'),
+    className: grid['col-xs-2'],
+    getValue: block => (
+      <Link to={`${routes.accounts.pathPrefix}${routes.accounts.path}/${block.generatorAddress}`}>
+        {block.generatorUsername}
+      </Link>
+    ),
+  }, {
+    header: t('Amount'),
+    className: grid['col-xs-1'],
+    getValue: block => <LiskAmount val={block.totalAmount} token={t('LSK')} />,
```

Comment:
`LSK` is the name of our token and it is the same for all languages. Please get the value of that form the token constant.

Reply:
Fixed. Please check again @yasharAyari